src/frontend/README.md

```diff
@@ -9,11 +9,17 @@ else.
 
 ## OpenTelemetry features
 
-| Signal  | Instrumentation library | Custom spans | Custom span attributes |
-|---------|-------------------------|--------------|------------------------|
-| Traces  | X                       | O            | X                      |
-| Metrics | O                       | na           | na                     |
-| Logs    | O                       | na           | na                     |
+| Traces                                    | Implemented | Metrics                                   | Implemented | Logs | Implemented |
+|-------------------------------------------|-------------|-------------------------------------------|-------------|------|-------------|
+| Automatic Instrumentation (Agent)         | n/a         | Automatic Instrumentation (Agent)         | n/a         |      |             |
```

Comment:
Not blocking:
1. Consider adding a link to each item with a concrete definition - e.g. what exactly does "Automatic Instrumentation (Agent)" mean.
2. Given the description + link might be long, consider changing the layout:

### Traces

* [ ] ⛔ [Automatic Instrumentation (Agent)](link)
* [ ] 🚧 ...

### Metrics

* [ ] ...

Reply:
Nice. I like this approach

Reply:
I updated the content using this structure. Not sure if the links are all perfect but it was a bit difficult to find the exact info documented.